core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala

```diff
@@ -54,7 +54,7 @@ import scala.collection.mutable.Buffer
 
 class AuthorizerIntegrationTest extends BaseRequestTest {
 
-  override def numBrokers: Int = 1
+  override def serverCount: Int = 1
```

Comment:
nit: we're a little inconsistent with naming (broker vs server). Perhaps this should be `brokerCount` to go along with `brokerPropertyOverrides`?